pkg/api/persistentvolumeclaim/util_test.go

```diff
@@ -269,6 +270,7 @@ func TestDataSourceFilter(t *testing.T) {
 
 	for testName, test := range tests {
 		t.Run(testName, func(t *testing.T) {
+			featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.32"))
```

Comment:
Add a note:
// TODO: this will be removed in 1.36

Reply:
Done, all updated